lib/tasks/cache_historical_data.rake

```diff
@@ -0,0 +1,16 @@
+desc "This task is run by a scheduling tool nightly to cache the processor intensive queries"
+task :cache_historical_data => :environment do
+  puts "Caching historical data"
```

Comment:
Should we be using `Rails.logger` here instead of `puts`?

Reply:
Switched to logger with `info`